<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1530</width>
    <height>830</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>1000</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Symbol</family>
    <pointsize>35</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::FocusPolicy::NoFocus</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>wellcome</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="font">
       <font>
        <family>W_adib</family>
        <pointsize>40</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#page {
  background-color: lightblue; /* رنگ پس‌زمینه */
  background-image:  url(:/new/prefix1/bc.png); /* تصویر پس‌زمینه */
  background-repeat: no-repeat; /* جلوگیری از تکرار تصویر */
  background-position: center; /* موقعیت تصویر */
}</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="wellcome">
       <property name="maximumSize">
        <size>
         <width>1508</width>
         <height>732</height>
        </size>
       </property>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>610</y>
          <width>171</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Sitka</family>
          <pointsize>28</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #A9F08E, stop: 1 #6AD84F);
    color: black;
    border: 3px solid #3F5931;
    border-radius: 15px;
    padding: 5px;
    box-shadow: 0px 2px 5px rgba(0, 0, 0, 0.3); /* سایه بیرونی */
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #B7D99D, stop: 1 #74B158);
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #6AA84F, stop: 1 #548235);
    box-shadow: inset 0px 1px 3px rgba(0, 0, 0, 0.4); /* سایه داخلی */
}</string>
        </property>
        <property name="text">
         <string>play</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page">
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>370</y>
          <width>281</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>W_adib</family>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>170</y>
          <width>241</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>W_adib</family>
          <pointsize>40</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>player1</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>1040</x>
          <y>170</y>
          <width>251</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>W_adib</family>
          <pointsize>40</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>player2</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>980</x>
          <y>360</y>
          <width>291</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>W_adib</family>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>300</y>
          <width>241</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>W_adib</family>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>enter your name:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>980</x>
          <y>300</y>
          <width>241</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>W_adib</family>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>enter your name:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_1">
        <property name="geometry">
         <rect>
          <x>1410</x>
          <y>690</y>
          <width>83</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>next</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="game">
       <widget class="QGraphicsView" name="game_2">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>120</y>
          <width>1101</width>
          <height>581</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
